deploy/kubernetes/overlays/stable/kustomization.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 bases:
 - ../../base
+patches:
+- controller_disable_topology.yaml
```

Comment:
I would prefer that all patches are additive to minimize confusion.
The way I think dependencies should work:
```
         -> stable
base -> alpha (add topology here) -> dev
                                                       -> staging
```
Let me know what you think

Reply:
sure that works

Reply:
pushed a new commit